spec/JSON-RPC.md

```diff
@@ -918,6 +923,104 @@ Response Example
 }
 ```
 
+## account_getAccountList
+Get a list of accounts.
+
+Params: No parameters
+
+Return Type: `H160[]`
+
+Request Example
+```
+curl \
+    -H 'Content-Type: application/json' \
+    -d '{"jsonrpc": "2.0", "method": "account_getAccountList", "params": [], "id": 6}' \
+    localhost:8080
+```
+
+Response Example
+```
+{
+  "jsonrpc":"2.0",
+  "result":["0x318def87d8dc0f7cc21794daf2dd36762db22b67"],
+  "id":6
+}
+```
+
+## account_createAccount
+Creates a new account.
```

Comment:
Creates -> Create

Reply:
`-s` is used here for consistency with others.